### DISCUSSION TITLE
Add 24h support to time input component @mantine/dates

### DISCUSSION THREAD

**Discussion by mehdiraized** [mantinedev/mantine#6080] [Feature requests]
Currently, the time input component likely supports the standard 12-hour time format (AM/PM), where the user can input time values like "1:00 PM" or "11:30 AM". However, with this request, the user is suggesting that the component should also support inputting time in the 24-hour format, commonly used in military, aviation, and some international contexts, where times are represented as "13:00" for 1:00 PM and "23:30" for 11:30 PM.

Implementing this feature would allow users of the library to input time values in both the 12-hour and 24-hour formats, providing more flexibility and accommodating users who prefer or are accustomed to using the 24-hour time format.

**Comment by zanehankin** [comment]
Please also add a "clearable" parameter

**Comment by holm-dk** [comment]
Did anyone ever implement a workaround?

**Comment by rtivital** [comment]
This feature is supported in TimePicker component starting from 8.0
